### PR TITLE
Done

### PR DESCRIPTION
**Pull request by umohm1**

### CODE REVIEW

**Comment by bhollan** [comment]
Hi @umohm1 !

Looking through your changes here, I see your CSS rules you added, and those look great.  But it looks like you left something at the top of `styles.css` that probably shouldn't be there.  The other thing I don't see is a reference to your profile in `index.html`.  If you can take care of those two small things, I think we can merge your changes in.  Thanks!

CC: @aturkewi

**Comment by umohm1** [comment]
Good Morning,

I've made the changes.

Thank you

-Mary

File: index.html [hunk 1]
```diff
@@ -23,6 +23,19 @@
         </div>
       </div>
 
+      <div class="roster-cards-container">
```
Comment (bhollan) [inline index.html]: There needs to be only one line like this.  I think this was a simple copy/paste accident.

Just delete this line and the rest of `index.html` should be ok.

File: css/styles.css [hunk 2]
```diff
@@ -1149,6 +1148,14 @@ a:visited {
  background-image: url("https://learn-verified.s3.amazonaws.com/marsi-meyers-miyashita.jpg");
 }
 
+#mary-umoh-cover {
+ background-image: url(http://i.imgur.com/7edFHon.jpg);
```
Comment (bhollan) [inline css/styles.css]: The URL here should have `"`s around it.  Take a look at the other student's CSS rules to see how it should look.

File: css/styles.css [hunk 3]
```diff
@@ -1149,6 +1148,14 @@ a:visited {
...
+}
+
+#mary-umoh-card {
+ background-image: url(http://i.imgur.com/zhFh0lO.jpg);
```
Comment (bhollan) [inline css/styles.css]: Same thing here.

**Comment by bhollan** [comment]
CC: @aturkewi

**Comment by bhollan** [comment]
Hi @umohm1 !

I'm going to close this for now.  If you get a chance to look at it and want to open it up again, that's fine.  Feel free to Ask A Question if you need any help!

CC: @aturkewi

**Pull request closed without merging**